### ISSUE TITLE
[Programmatic Access – Make Code - Footer] Incorrect role is defined for 'Language & Version' button controls present under ‘Footer’ section.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3520]
**User Experience:**
Users who depends upon screen reader get confused if the Screen reader reads incorrect role of any control. They'll not be able to know whether it is a link or a button or whether it is a non-interactive control. Users will understand it to be a interactive control as screen reader announces link while navigating in scan mode and hence it will not be a good usable experience for the users. 

**Note:** User credentials should NOT be included in the bug. 

**Pre-Requisite:**
Open F12 Developer tool.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify that incorrect role is defined for 'Language & Version' button controls or not.

**Actual Result:**
Incorrect role 'Link' is defined for 'Language & Version' button controls under ‘Footer’ section.

**Expected Result:**
Correct role should be define as 'Button' instead of link for 'Language & Version' button controls under ‘Footer’ section.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**WCAG Reference:**
[WCAG 4.1.2 – Name, Role, Value ](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.1.2, #Win10-Edge(Chromium)

![WCAG4 1 2_Incorrect role is defined](https://user-images.githubusercontent.com/72871089/97663561-f35cce80-1a9f-11eb-8f39-f7f6c6291528.jpg)
